### PR TITLE
update server.js

### PR DESCRIPTION
**Pull request by huwng31**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log("this is server");
```
Comment (nguyenxuanhung03101999) [inline server.js:1]: not good

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+.history
```
Comment (nguyenxuanhung03101999) [inline .gitignore:1]: not good

**Review by nguyenxuanhung03101999 — CHANGES_REQUESTED** [review]

**Review by nguyenxuanhung03101999 — COMMENTED** [review]
request change

**Review by nguyenxuanhung03101999 — COMMENTED** [review]
request change 2

File: server.js (hunk 1 above)
Comment (huwng31) [reply server.js:1]: Im fixed

Comment (nguyenxuanhung03101999) [reply server.js:1]: done

File: .gitignore (hunk 2 above)
Comment (huen555) [reply .gitignore:1]: Im fixed

Comment (nguyenxuanhung03101999) [inline .gitignore:1]: oke

Comment (huwng31) [reply .gitignore:1]: Im fixed

Comment (huen555) [reply .gitignore:1]: hehe

**Review by nguyenxuanhung03101999 — APPROVED** [review]
all good

**Pull request merged**